### ISSUE TITLE
Removing recipes on the rockbreaker causes error

### ISSUE THREAD

**Issue by LuckyLMJ** [GregTechCEu/GregTech#1266]
### GregTech CEu Version

2.4.3-beta

### Environment

Singleplayer

### New Worlds

Yes

### Cross-Mod Interaction

Yes

### Expected Behavior

Expected to remove the recipe on the rockbreaker.

### Actual Behavior

Threw the error
`
Error: [crafttweaker]: Error executing {[0:crafttweaker]: rockBreakerRecipes.zs}: gregtech/integration/GroovyScriptCompat, caused by java.lang.NoClassDefFoundError:
gregtech/integration/GroovyScriptCompat
`

### Steps to Reproduce

1. Make a crafttweaker script with the contents 
```
import mods.gregtech.recipe.RecipeMap;
val rockbreaker as RecipeMap = <recipemap:rock_breaker>;
rockbreaker.findRecipe(240, [<minecraft:redstone>], null).remove();
```
2. Launch the game

3. CraftTweaker throws an error

### Additional Information

This bug did not happen in 2.4.1-beta.

**Comment by tmtvl** [comment]
It should be fixed in 2.4.4, give it a try.

**Comment by LAGIdiot** [comment]
This was fixed in 2.4.4. please upgrade to new version.
